inst/input/hector_rcp26.ini

```diff
@@ -144,6 +144,7 @@ baseyear=1750		; when to start reporting; by definition, all F=0 in this year
 ; Optional radiative forcing constraint
 ; If supplied, the model will use these data, ignoring what it calculates
 ;Ftot_constrain=csv:constraints/MAGICC_RF_8.5.csv
+aco2=5.35		; radiative forcing, W/m2
```

Comment:
No I don't think so, I think that this would also be forcing efficiency but it is used to calculate the forcing per CO2 doubling that is used in the temperature component.